sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -1644,6 +1644,13 @@ object RapidsConf {
     .booleanConf
     .createWithDefault(value = true)
 
+  val FALLBACK_DELTA_CHECKPOINT = conf("spark.rapids.sql.fallbackDeltaCheckpoint")
```

Comment:
Curious why this isn't part of `detectDeltaLogQueries` and is a separate config?  Arguably this behavior is still delta log related, just happens to be parquet instead of json.  If we do still need a separate config, should it be `detectDeltaCheckpointQueries` to be consistent with the existing config?

Reply:
I put it under a separate config in case we found some case we wanted to do it separate from json.  Like our parquet reader is fast enough to make up for difference.  I can rename it to be consistent